app/models/user.rb

```diff
@@ -13,11 +12,11 @@ def has_obituary?(obituary)
   def self.create_with_omniauth(auth)
     where(email: auth.info.email).first_or_initialize do |user|
       user.fbid = auth.uid
-      user.first_name = auth.info.first_name 
-      user.last_name = auth.info.last_name 
-      user.email = auth.info.email 
+      user.first_name = auth.info.first_name
+      user.last_name = auth.info.last_name
```

Comment:
Are we still planning to combine `first_name` and `last_name` in the db to just be `name`?

Reply:
I think for the user it makes sense to have first and last name separate, but for the obituaries it would make our lives a lot easier to just have a name field and when we search, we search the whole name rather than splitting it up. Also, as you know, the NYT data just has name and doesn't split them up.

Reply:
Sounds good